### PR TITLE
Fix LIBTD-79: Updating a collection will auto-update the metadata of …

### PR DESCRIPTION
**Pull request by tingtingjh**
…its corresponding DOI through EZID

### CODE REVIEW

File: app/controllers/collections_controller.rb [hunk 1]
```diff
@@ -53,6 +53,25 @@ def update
         end
       end
       @collection.update_attributes({:funder => newfunder})
+      @collection[:identifier].each do |id|
+        if /\Adoi:10.5072\/FK2/ =~ id
```
Comment (rotated8) [inline app/controllers/collections_controller.rb]: Can you add a comment about what this regex is trying to match?

Comment (yinlinchen) [inline app/controllers/collections_controller.rb]: Isn’t that straightforward?

Comment (tingtingjh) [inline app/controllers/collections_controller.rb]: Something like this?

# if this dataset has an existing identifier which is a DOI assigned by VT libraries through EZID;
# that is, it is started with the unique prefix associated with VT libraries 
if /\A#{Ezid::Client.config.default_shoulder}/ =~ id

Comment (rotated8) [inline app/controllers/collections_controller.rb]: Yes, or `#Matches DOIs minted by VT Libraries.`

Comment (tingtingjh) [inline app/controllers/collections_controller.rb]: Something like the following?
# if this dataset has an existing identifier which is a DOI assigned with VT libraries through EZID;
# that is, it is started with the unique prefix associated with VT libraries
if /A#{Ezid::Client.config.default_shoulder}/ =~ id

Comment (rotated8) [inline app/controllers/collections_controller.rb]: @yinlinchen I don't think so. 'doi:10.xxxx' doesn't inherently mean anything, unless you know only DOIs we've minted start with that. It looks like a magic string to me.

**Pull request merged**